content/04.network_applications.md

```diff
@@ -67,30 +67,31 @@ Finally, each vector is optimized while preserving the global equation: $\textbf
 
 Deepwalk is an early method that represents knowledge graphs in a low dimensional space [@doi:10.1145/2623330.2623732]. 
 The first step of this method is to perform a random walk along a knowledge graph.
 During the random walk, every generated sequence of nodes is recorded and treated as a sentence in word2vec [@arxiv:1310.4546; @arxiv:1301.3781].
 After every node has been processed, a skip-gram model is trained to predict the context of each node thereby constructing a low dimensional representation of a knowledge graph [@doi:10.1145/2623330.2623732].
 A limitation for deepwalk is that the random walk cannot be controlled, so every node has an equal chance to be reached.
 Grover and Leskovec demonstrated that this limitation can hurt performance when classifying edges between nodes and developed node2vec as a result [@arxiv:1607.00653].
-Node2vec operates the in the same fashion as deepwalk; however, this algorithm specifies a parameter that lets the random walk be biased when traversing nodes [@arxiv:1607.00653].
+Node2vec operates in the same fashion as deepwalk; however, this algorithm specifies a parameter that lets the random walk be biased when traversing nodes [@arxiv:1607.00653].
 A caveat to both deepwalk and node2vec is that they ignore information such as edge type and node type.
 Various approaches have evolved to fix this limitation by incorporating  node, edge and even path types when representing knowledge graphs in a low dimensional space [@doi:10.1145/3097983.3098061; @doi:10.1145/3097983.3098036; @arxiv:1809.02269; @arxiv:1808.05611].
 An emerging area of work is to develop approaches that capture both the local and global structure of a graph when constructing this low dimensional space.
 
-Instead of using the word2vec framework, some deep learning approaches use an adjacency matrix as input [@arxiv:1310.4546; @arxiv:1301.3781].
-These approaches use neural networks called autoencoders to generate this low dimensional space [@doi:10.1109/DSAA.2018.00034; @arxiv:1611.07308; @arxiv:1802.04407].
-Autoencoders map input such as an adjacency matrices into a low dimensional space and then determines how to construct this space via reconstructing the same input [@doi:10.1016/j.neunet.2014.09.003; @raw:Baldi2011].
-This generated space represents the nodes and their connectivity structure within a knowledge graph [@doi:10.1109/DSAA.2018.00034; @arxiv:1611.07308; @arxiv:1802.04407].
+Instead of using the word2vec framework, some neural network approaches use an adjacency matrix as input [@arxiv:1310.4546; @arxiv:1301.3781].
+These approaches use models called autoencoders [@doi:10.1109/DSAA.2018.00034; @arxiv:1611.07308; @arxiv:1802.04407].
+Autoencoders are designed to map input into a low dimensional space and learns how to construct this space by reconstruction of the same input [@doi:10.1016/j.neunet.2014.09.003; @raw:Baldi2011].
```

Comment:
```suggestion
Autoencoders are designed to map input into a low dimensional space and then back to a reconstruction of the same input [@doi:10.1016/j.neunet.2014.09.003; @raw:Baldi2011].
It is possible to layer on additional objectives by modifying the loss function to take into account criteria above and beyond reconstruction loss.
```

Reply:
Up to you if you want that second sentence. It lets you note something that's probably important though maybe not in this domain. Could cite VAEs as an example (Autoencoding variational bayes is the paper title - someone must have done a VAE of a graph too).